### DISCUSSION TITLE
No support for (edited)

### DISCUSSION THREAD

**Discussion by opsecx** [community/community#109373] [Mobile] [answered]
<details>
<summary>ℹ️ Not sure why you would drop support for (edited), is this a conscious decision?</summary>
</details>

**Comment by iskae** [comment, marked as answer]
Hi @opsecx 👋 

Thank you for reaching out ❤️ Unfortunately, yes, this was a conscious decision from our side 😅 We are sorry that this decision affected you personally and it was not an easy decision to take for us too 🙇 This will allow us to build more stable apps for the future.

**Reply by opsecx** [reply]
Hi, thanks for your reply. I understand at some point support for earlier versions have to be dropped, but as (edited) still receives updates from apple, it feels a bit premature? Of course I'm biased :)

**Reply by Ayzzzzzzzzzzz** [reply]
yes agreed
